parsers.go

```diff
@@ -85,6 +88,22 @@ func RouterLogParser(msg []byte) *LogMsg {
 	return m
 }
 
+func YellowfinLogParser(msg []byte) *LogMsg {
+	matches := yellowfinLogRegex.FindSubmatchIndex(msg)
+	if len(matches) != (4+1)*2 {
+		return nil
+	}
+	var err error
+	m := &LogMsg{}
+	m.Time, err = time.ParseInLocation(timeYellowfin, string(getCapture(msg, matches, 1)), time.Now().Location())
```

Comment:
Instead of `time.Now().Location()`, use `time.Local` to retrieve the local timezone.